app/Http/Controllers/DashboardController.php

```diff
@@ -6,10 +6,13 @@
 use App\Model\Order\Invoice;
 use App\Model\Order\Order;
 use App\Model\Payment\Currency;
+use App\Model\Product\Product;
 use App\Model\Product\Subscription;
 use App\User;
 use Carbon\Carbon;
+use DateTime;
 use Illuminate\Http\Request;
+use phpDocumentor\Reflection\Types\Collection;
 
 class DashboardController extends Controller
```

Comment:
Can you check Create new User button url on the dashboard by clicking on it.

Reply:
Also if the admin logs in, do we need to change the url to /dashboard?

Reply:
> Also if the admin logs in, do we need to change the url to /dashboard?

laravel files has been changed. We will do it when we correct those

Reply:
> Please check Create new User button url on the dashboard by clicking on it.

fixed